app/graphql/types/book_attributes.rb

```diff
@@ -1,15 +1,15 @@
 module Types
   class BookAttributes < BaseInputObject
     description "Attributes for creating or updating a Book"
-    argument :google_book_id, String, required: true
-    argument :isbn_13, String, required: true
-    argument :pg_count, Integer, required: true
-    argument :description, String, required: true
-    argument :author, String, required: true
-    argument :book_title, String, required: true
-    argument :book_cover, String, required: true
-    argument :category, String, required: true
-    argument :condition, String, required: true
-    argument :available, Boolean, required: true
+    argument :google_book_id, String, required: false
+    argument :isbn_13, String, required: false
+    argument :pg_count, Integer, required: false
+    argument :description, String, required: false
+    argument :author, String, required: false
+    argument :book_title, String, required: false
+    argument :book_cover, String, required: false
+    argument :category, String, required: false
+    argument :condition, String, required: false
+    argument :available, Boolean, required: false
   end
```

Comment:
Thank you so much for bringing up this idea! Wild how much more simple it was than I thought